documentation/Quickstart-on-how-to-write-tests.md

```diff
@@ -133,6 +137,79 @@ If we have existing recordings then the tests have been run against generated th
 > export TEST_MODE=playback
 > rushx test
 ```
+## How to Push Test recordings to Assets repo
+We need to push test recording files to [asset repo](https://github.com/Azure/azure-sdk-assets) after testing your test cases.
+
+### Push test recording
+If your package is new without any recordings, go to the next section [New Package - No Recorded Tests](#new-package---no-recorded-tests)
+
+If you already have an assets.json file in your package root directory, go to [Existing package - Tests have been pushed before](#existing-package---tests-have-been-pushed-before).
+
+#### New Package - No Recorded Tests
+This section assumes that your package is new to the JS repo and that you're trying to onboard your tests with recorder, and the asset-sync workflow.
+
+Generate an `sdk/<service-folder>/<package-name>/assets.json` file by running the following command.
+
+```bash
+npx dev-tool test-proxy init
```

Comment:
@HarshaNalluru Do we have a command which could help 
1) init the assert environment or 2) do nothing if assert.json is there?

If so I was wondering we could directly add the command before customers run any testing.

Another point is if we could append the `push` command after the testing finishs in our package.json so customers don't need to know any push actions.

Reply:
> Just curious do we have a command which could help... 1. init the assert environment or 2) do nothing if assert.json is there?
We could modify init to do nothing if assert.json is already present.

> Another point is if we could append the push command after the testing finishs in our package.json so customers don't need to know any push actions.
We want the users/partners to be aware of the recordings and view them before pushing them to make sure they are as expected.
Automating that might defeat that purpose.

Reply:
Sure let's add these steps to let customers run first and open to collect any feedback.